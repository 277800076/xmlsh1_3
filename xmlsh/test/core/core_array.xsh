# core_array.xsh
# test array notation

a=()
echo ${#a}
a=(foo)
echo ${#a}
a=(foo bar spam)
echo ${#a}
echo ${a[2]}
echo ${a[*]}

a=<[ ('foo' , <spam><bar a="attr">text</bar></spam>, 1) ]>
set +indent
xecho ${a[2]}

	 