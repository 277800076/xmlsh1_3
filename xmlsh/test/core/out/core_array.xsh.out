0
1
3
bar
foo bar spam
<spam><bar a="attr">text</bar></spam>
